contracts/pToken.sol

```diff
@@ -76,6 +77,11 @@ contract PToken is
             "Recipient cannot be the token contract address!"
         );
         _mint(recipient, value, userData, operatorData);
+        if (userData.length > 0) {
+            // TODO: check if this is needed, and, eventually, replace .code with a function
```

Comment:
Is it required? Right now there's an exception thrown, i.e.
```
Transaction reverted: function call to a non-contract account
```
(check the added test).